healthcareai/tests/test_oversampling.py

```diff
@@ -0,0 +1,58 @@
+import unittest
```

Comment:
Please run a linter (pylint is easy) to ensure the formatting is consistent. If you need guidance, I'm happy to assist.